Doc/library/importlib.rst

```diff
@@ -1297,30 +1297,27 @@ find and load modules.
 
    (``__name__``)
 
-   A string for the fully-qualified name of the module.
+   Fully-qualified name of the module.
 
    .. attribute:: loader
 
    (``__loader__``)
 
-   The loader to use for loading.  For namespace packages this should be
-   set to ``None``.
+   Loader object to use for loading the module.
 
    .. attribute:: origin
 
    (``__file__``)
 
-   Name of the place from which the module is loaded, e.g. "builtin" for
-   built-in modules and the filename for modules loaded from source.
-   Normally "origin" should be set, but it may be ``None`` (the default)
-   which indicates it is unspecified (e.g. for namespace packages).
+   Path to the code from where the module should be loaded.
+   It is ``None`` for builtin modules and namespace package modules.
 
    .. attribute:: submodule_search_locations
 
    (``__path__``)
 
-   List of strings for where to find submodules, if a package (``None``
-   otherwise).
+   Iterable (possibly empty) of locations to search for submodules of a
+   package module during import. It is `None` for non-package modules.
```

Comment:
"package during import", i.e. I don't think you need to say "package module" here.

Reply:
Done.